compiler/src/test/java/io/neow3j/compiler/ContractInterfaceTest.java

```diff
@@ -83,7 +83,7 @@ public void testConstantStringParam() throws IOException {
                 is(hexStringToByteArray(reverseHexString("ef4073a0f2b305a38ec4050e4d3d28bc40ea63f5")))); // hash
         assertThat(insns.get(22).getOpcode(), is(OpCode.NEWARRAY0)); // array, hash
         assertThat(insns.get(23).getOpcode(), is(OpCode.SWAP)); // hash, array
-        assertThat(insns.get(24).getOpcode(), is(OpCode.PUSH15)); // callflags, hash, array
+        assertThat(insns.get(24).getOpcode(), is(OpCode.PUSH5)); // callflags, hash, array
```

Comment:
😱 😱 😱 😱 😱 😱 😱 😱 
what's the reason for this change?

Reply:
oh, ok, this is the test. Sorry, this is just to accommodate the changes. All clear.